### PR TITLE
Update Text-Turkish-1055.txt

### PR DESCRIPTION
**Pull request by fmbxnary**

### CODE REVIEW

**Review by isaak654 — CHANGES_REQUESTED** [review]
Something failed with the changes in https://github.com/sandboxie-plus/Sandboxie/pull/2036/commits/7d2a50cee7d6b2780f80443ae92cd25f023c65b5

File: Sandboxie/msgs/Text-Turkish-1055.txt [hunk 1]
```diff
@@ -2520,6 +2397,27 @@ Tamam tıklayarak '%2' sandbox'ı içindeki Bırakılan Haklar ayarını kapatı
...
+3955;txt;01
+Ağ Dosyaları
+.
+
+3956;txt;01
+Ağ dosyaları ve klasörleri normalde korumalı alan uygulamaları tarafından görülebilir.
+.
+
+3957;txt;01
+
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Turkish-1055.txt:2413]: I would suggest to remove this line to resolve the conflict.

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Sandboxie/msgs/Text-Turkish-1055.txt [hunk 2]
```diff
@@ -3203,6 +3175,58 @@ Yeniden adlandı&r
...
+6003;txt;01
+Sandboxie classic'in eski kullanıcı arayüzü, ağ güvenlik duvarı işlevi için destek uygulamaz.
+
+Çekirdek bileşenler tarafından desteklenen özelliği ini dosyasını kullanarak yapılandırabilirsiniz, kurallar aşağıdaki gibi yapılandırılmıştır:
+NetworkAccess=iexplorer.exe,İzin Ver; bağlantı noktası=80,443; Adres=192.168.0.1-192.168.100.255; Protokol=TCP
+
+Bu seçenekleri kontrol etmek için bir UI kullanmayı tercih ederseniz, lütfen tüm yeni özellikler için tam UI desteğine sahip Sandboxie-Plus'a yükseltin.
+.
+
+6004;pop;hata;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Turkish-1055.txt:3207]: It should be `6004;pop;err;01`

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Sandboxie/msgs/Text-Turkish-1055.txt (hunk 2 above)
Comment (fmbxnary) [reply Sandboxie/msgs/Text-Turkish-1055.txt]: ahh thank you. This solved the problem

**Review by isaak654 — APPROVED** [review]

**Pull request merged**